Source/ACE/Entity/Player.cs

```diff
@@ -456,6 +478,42 @@ public void Load(AceCharacter character)
             Session.Network.EnqueueSend(setTurbineChatChannels, general, trade, lfg, roleplay);
         }
 
+        public AceObject GetSavableCharacter()
+        {
```

Comment:
So this is a code smell, too.  AceObject can never behave as an AceCharacter.  What's the use case here?